src/test/java/com/productdock/library/search/KafkaConsumerTest.java

```diff
@@ -0,0 +1,41 @@
+package com.productdock.library.search;
+
+
+import com.productdock.library.search.cosumer.messages.InsertBook;
+import com.productdock.library.search.data.provider.KafkaTestProducer;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.time.Duration;
+
+import static com.productdock.library.search.data.provider.InsertBookMessageMother.defaultInsertBookMessageBuilder;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+@SpringBootTest
+public class KafkaConsumerTest extends IntegrationTestBase {
```

Comment:
Too generic test name, it should reflect what it actually tests. IndexingNewBookTest maybe